src/components/ToastTextToCad.tsx

```diff
@@ -344,15 +353,30 @@ export function ToastTextToCadSuccess({
               }
               if (isDesktop()) {
                 // Delete the file from the project
-                fileMachineSend({
-                  type: 'Delete file',
-                  data: {
-                    name: data.fileName,
-                    path: `${context.project.path}${window.electron.sep}${data.fileName}`,
-                    children: null,
-                  },
-                })
+
+                if (projectName && fileName) {
+                  // You are in the new workflow for text to cad at the global application level
```

Comment:
```suggestion
                  // You are in the workflow for text to cad at the global application level
```

Nit: this comment's gonna get out-of-date real quick.